service/tcp.go

```diff
@@ -143,6 +146,10 @@ type TCPService interface {
 	GracefulStop() error
 }
 
+func (s *tcpService) SetTargetIPValidator(isTargetIPAllowed func(ip net.IP) *onet.ConnectionError) {
```

Comment:
We have a type for this: `onet.IPPolicy`.  I think we might as well use it.

Reply:
Done